Fail to install (Peer babel-plugin-espower@0.3.1 wants babel-core@>=5.2.0 <5.4.3)

Try to install `espower-babel` and failed.

Reproduce step:

```sh
mkdir tmp
cd tmp
npm init --yes
npm install -S espower-babel
```

Output:
```
npm WARN package.json tmp@1.0.0 No description
npm WARN package.json tmp@1.0.0 No repository field.
npm WARN package.json tmp@1.0.0 No README data
npm WARN peerDependencies The peer dependency babel-core@>=5.2.0 <5.4.3 included from babel-plugin-espower will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/Users/azu/.nodebrew/current/bin/npm" "install" "-S" "espower-babel"
npm ERR! node v0.12.0
npm ERR! npm  v2.7.5
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package babel-core does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer babel-plugin-espower@0.3.1 wants babel-core@>=5.2.0 <5.4.3

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/azu/.ghq/github.com/azu/tmp/tmp/npm-debug.log
```

## Round up

[espower-babel](https://github.com/power-assert-js/espower-babel "espower-babel")'s dependencies

```js
  "dependencies": {
    "babel-core": "^5.0",
    "babel-plugin-espower": "^0.3.0"
   }
```

[babel-plugin-espower](https://github.com/power-assert-js/babel-plugin-espower/blob/4bf320c75e1642eb057bfa558d03ba0dc452241f/package.json "babel-plugin-espower")'s peerDependencies :

```js
  "peerDependencies": {
    "babel-core": ">=5.2.0 <5.4.3"
  }
```

These are conflicted.

- espower-babel require babel-core 5.x(It contain 5.4.3 >=)
- babel-plugin-espower require babel-core ">=5.2.0 <5.4.3"

As a result, try to install babel-core 5.4.3 >= and throw peerinvalid error.

--- a/index.js
+++ b/index.js
@@ -37,7 +37,7 @@ function espowerBabel(options) {
             return pluginName === "babel-plugin-espower";
         });
         if (!espowerPluginExists) {
-            babelOptions.plugins.push(createEspowerPlugin(options.espowerOptions));
+            babelOptions.plugins.push(createEspowerPlugin(babel, options.espowerOptions));
         }
         return babelOptions;
     }

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
   },
   "dependencies": {
     "babel-core": "^5.0",
-    "babel-plugin-espower": "^0.3.0",
+    "babel-plugin-espower": "^0.4.0",
     "minimatch": "^2.0.1",
     "source-map-support": "^0.2.10",
     "xtend": "^4.0.0"

